src/utils/transactions.ts

```diff
@@ -24,52 +24,18 @@ import {
 } from '@alephium/sdk'
 import { Output, Transaction, UnconfirmedTransaction } from '@alephium/sdk/api/explorer'
 
-import { Address } from '@/contexts/addresses'
-import { AddressHash } from '@/types/addresses'
+import { AddressHash, AddressRedux } from '@/types/addresses'
 import { NetworkName } from '@/types/network'
-import { PendingTx, TransactionStatus } from '@/types/transactions'
-
-export type TransactionVariant = Transaction | PendingTx
-type IsTransactionVariant<T extends TransactionVariant> = T extends Transaction
-  ? Transaction
-  : T extends PendingTx
-  ? PendingTx
-  : never
-export type BelongingToAddress<T extends TransactionVariant> = { data: IsTransactionVariant<T>; address: Address }
```

Comment:
I feel so clean we don't need this 💩 anymore!

Reply:
Oh yes.